eng/pipelines/templates/variables/vmr-stage.yml

```diff
@@ -28,6 +28,8 @@ variables:
   value: mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0-cross-arm64
 - name: androidCrossContainer
   value: mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0-cross-android-amd64
+- name: wasmCrossContainer
+  value: mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0-webassembly-20230917141449-2aaa02c
```

Comment:
I know we pinned this version because of some issue in runtime but we should get back to the floating tag ASAP, do you know if we have an issue?

Reply:
I don't know, I just pulled what was in my runtime checkout. I'll ask someone from the wasm team tomorrow

Reply:
https://github.com/dotnet/runtime/pull/92243 was the last time it was touched.  IIRC there was some issue with mariner and the libc for one of the wasm deps??